ui/app/AppLayouts/Chat/stores/MessageStore.qml

```diff
@@ -207,9 +207,6 @@ QtObject {
     }
 
     property bool playAnimation: {
```

Comment:
Should be `readonly` probably

Reply:
Ideally shouldn't be here at all :rofl: 
Added `readonly` for now.